ta/arch/arm/link.mk

```diff
@@ -73,6 +73,8 @@ link-script-cppflags-$(sm) := \
 
 link-script-pp-makefiles$(sm) = $(filter-out %.d %.cmd,$(MAKEFILE_LIST))
 
+TA_SIG_DIR ?= signatures
```

Comment:
Should be documented, somehting like:
`Path of the directory that stores file <uuid>.sig files each storing the signature to be stitched to <uuid>.elf file into a loadable signed <uuid>.ta image.`

Reply:
I will add a comment. Yes, all paths are relative to the top level directory of the project.

Reply:
> Path of the directory that stores file <uuid>.sig files each storing the signature to be stitched to <uuid>.elf file into a loadable signed <uuid>.ta image.

Aren't signatures added to the `<uuid>.stripped.elf` files?